.github/workflows/android-qa-tests.yml

```diff
@@ -0,0 +1,80 @@
+# This workflow needs to be kept at par with
+# https://github.com/simpledotorg/simple-android/blob/master/.github/workflows/ci_checks.yml
```

Comment:
This workflow is to run a QA integration test against a branch to catch regressions before releasing to master.

We had a version of this which was removed in https://github.com/simpledotorg/simple-server/pull/3287

Reply:
Ripping this out in favor of https://github.com/simpledotorg/simple-android/pull/4645/files